sched/scheduler/stateful_scheduler.go

```diff
@@ -116,15 +113,16 @@ func NewStatefulScheduler(
 		}()
 	}
 
+	// Recover Jobs in a separate go routine to allow the scheduler
```

Comment:
so right now we will accept and run new jobs as we recover.  There is a ticket to add back pressure ( not accept too many jobs ) so users would get a Scoot Too Busy error instead of waiting a long time. 

I agree it would be nice in the future to have some kind of estimated wait time, but that's a bit far off.